docs/SlavaUkraini/examples/overlays/index.md

```diff
@@ -0,0 +1,253 @@
+---
+layout: tutorial_v2
+title: Overlays
+---
+
+## Overlays
+
+There are 3 Overlays in Leaflet API: 
+- [`ImageOverlay`](/reference.html#imageoverlay): Raster Layer, Extends [`Layer`](/reference.html#layer)
+- [`VideoOverlay`](/reference.html#videooverlay): Raster Layer, Extends [`ImageOverlay`](/reference.html#imageoverlay)
+- [`SVGOverlay`](/reference.html#svgoverlay): Vector Layer, Extends [`ImageOverlay`](/reference.html#imageoverlay)
+
+In this tutorial, you’ll learn how to use these Overlays.
+
+### `ImageOverlay`
+
+`L.ImageOverlay` is used to load and display a single image over specific bounds of the map. 
+
+To add an image overlay [`L.ImageOverlay`](/reference.html#imageoverlay) use this:
+
+```
+var imageOverlay = L.imageOverlay(imageUrl, latLngBounds, options);
+```
+
+First of all, create a Leaflet map and add a background `L.TileLayer` in the usual way:
```

Comment:
The VideoOverlay section has a "Creating a map" h4 before the corresponding instruction, I suggest adding the same heading here.